docs/concepts/what-is-multi-cluster-replication.md

```diff
@@ -12,6 +12,15 @@ When necessary, for higher availability, Cluster operators can failover to any o
 
 Temporal's Multi-Cluster Replication feature is considered **experimental** and not subject to normal [versioning and support policy](/clusters).
 
+Temporal automatically forwards Start, Signal, and Query requests to the active Cluster.
+This feature must be enabled through a Dynamic Config flag per [Global Namespace](/concepts/what-is-a-global-namespace).
```

Comment:
Is "Dynamic Config" the name (or partial name) of the flag? I can't match it elsewhere in the docs.